tests/functional/clients/standalone/package_list/test_centos.py

```diff
@@ -7,11 +7,11 @@
 
 class TestRpm:
 
-    @pytest.mark.parametrize('path', path_params(centos.pkgfiles_all))
-    def test_pkgfiles_all(self, analyzed_data, path): #, path):
-        result = analyzed_data("rpm")
-        pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgfiles.all']['base']
-        assert pkgs.get(path) == 'RPMFILE'
+    # @pytest.mark.parametrize('path', path_params(centos.pkgfiles_all))
```

Comment:
are you meant to comment this out? or was this temporary only?

Reply:
why can't this be kept uncommented bith with a `skipif` ? The TODO message could be the `skipif` reason